packages/odyssey/src/scss/components/_button.scss

```diff
@@ -12,158 +12,156 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.ods-button {
-  display: inline-block;
-  position: relative;
-  margin: 0;
-  padding: $em-tiny-spacing $em-base-spacing;
-  transition-property: color, background-color, border-color, box-shadow;
-  transition-duration: 100ms;
-  transition-timing-function: linear;
-  border: 1px solid transparent;
-  border-radius: $base-border-radius;
-  background-color: cv('action');
-  box-shadow: 0 0 0 0 cv('action', 'light');
-  color: $white;
-  font-family: $body-font-family;
-  font-size: ms(0);
-  font-weight: 600;
-  line-height: $base-line-height;
+@mixin make-ods-button-variant($theme) {
```

Comment:
Still not sure it's worth keeping this. The mixin itself is almost as much work as redeclaring the button overrides - and this might output more CSS than is necessary?

Reply:
I agree its not worth keeping, and it definitely outputs more CSS than necessary. Need to clean up.